packages/feedback-service/service.ts

```diff
@@ -1,73 +1,72 @@
-import dotenv from 'dotenv-safe';
-if ( process.env.NODE_ENV === 'test' ) {
-  dotenv.config( { path: '.test.env' } );
-} else {
-  dotenv.config();
-}
-
```

Comment:
Add this env setup back.

Use this env setup instead of the dotenv-webpack, as this makes it easier to change the environment variables at deploy time, instead of at the build time.